api/handler/v1beta1/grpc.go

```diff
@@ -75,6 +75,7 @@ type appealService interface {
 	MakeAction(context.Context, domain.ApprovalAction) (*domain.Appeal, error)
 	Cancel(context.Context, string) (*domain.Appeal, error)
 	Revoke(ctx context.Context, id, actor, reason string) (*domain.Appeal, error)
+	BulkRevoke(ctx context.Context, filters *domain.ListAppealsFilter, actor, reason string) ([]*domain.Appeal, error)
```

Comment:
no, creating a new filter will add duplicate methods like s.repo.Find(listAppealFilter) and s.repo.Find(bulkRevokeFilter). So I am reusing the filter and repo method to filter appeals.

Reply:
instead, we can transform `BulkRevokeFilter` to `domain.ListAppealFilter` in appealService.BulkRevoke as well as adding the appeal status `active` to the list filter. Because it doesn't make sense to have appeal status filter in revoke since we can only revoke active appeals

Reply:
@rahmatrhd The [API contract](https://github.com/odpf/proton/blob/main/odpf/guardian/v1beta1/guardian.proto#L437) of bulk revoke doesn't take appeal status as a filter. Can you please suggest the changes.

Reply:
> The [API contract](https://github.com/odpf/proton/blob/main/odpf/guardian/v1beta1/guardian.proto#L437) of bulk revoke doesn't take appeal status as a filter. Can you please suggest the changes.

then it's better to have its own filter struct instead of using ListAppealFilter, the code later could be like this 
```go
func (s *service) BulkRevoke(ctx context.Context, filters *domain.RevokeAppealsFilter, actor, reason string) ([]*domain.Appeal, error) {
  appeals, err := s.repository.List(&domain.ListAppealsFilter{
    Status: 'active',
    ResourceURN: filters.ResourceURN,
    ResourceType: filters.ResourceType,
    ...
  })
  ...
}
```

Reply:
done